components/kernels/node.js

```diff
@@ -5,8 +5,11 @@ import Kernel from "./kernel";
 import SyntaxHighlighter from "react-syntax-highlighter";
 import { github } from "react-syntax-highlighter/styles/hljs";
 import { ContentSection } from "../content-section";
+import kernels from '../../pages/kernels';
 
-const install = `npm install -g ijavascript`;
+const install = `// From a terminal
```

Comment:
I think we can remove this comment since the text above it already clarifies it needs to be run from a template.

```suggestion
const install = `
```